packages/compass-databases-navigation/src/placeholder-item.tsx

```diff
@@ -38,9 +45,10 @@ export const PlaceholderItem: React.FunctionComponent<{
 }> = ({ type = 'collection', style }) => {
   const useNewSidebar = process?.env?.COMPASS_SHOW_NEW_SIDEBAR === 'true';
   const variant = useNewSidebar ? 'new' : 'old';
+  const { theme } = useTheme();
   return (
     <div className={cx(placeholderItem, padding[variant][type])} style={style}>
-      <Placeholder darkMode></Placeholder>
+      <Placeholder darkMode={theme === Theme.Dark}></Placeholder>
```

Comment:
Maybe would make sense to make this component "connected" to theme provider by default, similar to how we do this for lg components. Then you would not need to pass this here explicitly and can remove useTheme call

Reply:
good idea